templates/validator_stats_table.html

```diff
@@ -66,7 +66,6 @@ <h1 class="h4 mb-1 mb-md-0">
             <th title="Balance at the start of the day" data-toggle="tooltip">Start</th>
             <th title="Balance at the end of the day" data-toggle="tooltip">End</th>
             <th title="Amount of missed attestations" data-toggle="tooltip">M</th>
-            <th title="Amount of orphaned attestations" data-toggle="tooltip">O</th>
```

Comment:
**issue:** With this change the first and second line of the data table don't fit together anymore, e.g. the "Amount of proposed blocks" looks to be in the **Attestations** column.

Reply:
Ah yes thanks - totally did not see that first row ;-)